packages/Graph/GraphBotComponent.cs

```diff
@@ -31,6 +36,12 @@ public override void ConfigureServices(IServiceCollection services, IConfigurati
                     services.AddSingleton<DeclarativeType>(sp => new DeclarativeType(attribute.DeclarativeType, type));
                 }
             }
+
+            services.AddSingleton<JsonConverterFactory, JsonConverterFactory<ObjectExpressionConverter<CalendarSkillEventModel>>>();
+            services.AddSingleton<JsonConverterFactory, JsonConverterFactory<ObjectExpressionConverter<OrdinalV2>>>();
+            services.AddSingleton<JsonConverterFactory, JsonConverterFactory<ObjectExpressionConverter<DateTime?>>>();
+            services.AddSingleton<JsonConverterFactory, JsonConverterFactory<ArrayExpressionConverter<Attendee>>>();
```

Comment:
Should we pre-emptively add an object expression converter for Attendee as well?